website/docs/reference/resource-configs/teradata-configs.md

```diff
@@ -0,0 +1,212 @@
+      )
+    }}
+    ```
+    ```yaml
+    {{
+      config(
+          materialized="table",
+          table_option="NO FALLBACK, NO JOURNAL, CHECKSUM = ON,
+            NO MERGEBLOCKRATIO,
+            WITH CONCURRENT ISOLATED LOADING FOR ALL"
+      )
+    }}
+    ```
+  For details, see [CREATE TABLE documentation](https://docs.teradata.com/r/76g1CuvvQlYBjb2WPIuk3g/B6Js16DRQVwPDjgJ8rz7hg).
+* `with_statistics` - should statistics be copied from the base table, e.g.:
+    ```yaml
+    {{
+      config(
+          materialized="table",
+          with_statistics="true"
+      )
+    }}
+    ```
+    For details, see [CREATE TABLE documentation](https://docs.teradata.com/r/76g1CuvvQlYBjb2WPIuk3g/B6Js16DRQVwPDjgJ8rz7hg).
+
+* `index` - defines table indices:
+    ```
+    [UNIQUE] PRIMARY INDEX [index_name] ( index_column_name [,...] ) |
+    NO PRIMARY INDEX |
+    PRIMARY AMP [INDEX] [index_name] ( index_column_name [,...] ) |
+    PARTITION BY { partitioning_level | ( partitioning_level [,...] ) } |
+    UNIQUE INDEX [ index_name ] [ ( index_column_name [,...] ) ] [loading] |
+    INDEX [index_name] [ALL] ( index_column_name [,...] ) [ordering] [loading]
+    [,...]
+    ```
+    where:
+    * partitioning_level:
+      ```
+      { partitioning_expression |
+        COLUMN [ [NO] AUTO COMPRESS |
+        COLUMN [ [NO] AUTO COMPRESS ] [ ALL BUT ] column_partition ]
+      } [ ADD constant ]
+      ```
+    * ordering:
+      ```
+      ORDER BY [ VALUES | HASH ] [ ( order_column_name ) ]
+      ```
+    * loading:
+      ```
+      WITH [NO] LOAD IDENTITY
+      ```
+    e.g.:
+    ```yaml
+    {{
+      config(
+          materialized="table",
+          index="UNIQUE PRIMARY INDEX ( GlobalID )"
+      )
+    }}
+    ```
+    :information_source: Note, unlike in `table_option`, there are no commas between index statements!
+
+    ```yaml
+    {{
+      config(
+          materialized="table",
+          index="PRIMARY INDEX(id)
+          PARTITION BY RANGE_N(create_date
+                        BETWEEN DATE '2020-01-01'
+                        AND     DATE '2021-01-01'
+                        EACH INTERVAL '1' MONTH)"
+      )
+    }}
+    ```
+    :information_source: Note, unlike in `table_option`, there are no commas between index statements!
+
+    ```yaml
+    {{
+      config(
+          materialized="table",
+          index="PRIMARY INDEX(id)
+          PARTITION BY RANGE_N(create_date
+                        BETWEEN DATE '2020-01-01'
+                        AND     DATE '2021-01-01'
+                        EACH INTERVAL '1' MONTH)
+          INDEX index_attrA (attrA) WITH LOAD IDENTITY"
+      )
+    }}
+    ```
+## Seeds
+
+* `use_fastload` - use [fastload](https://github.com/Teradata/python-driver#FastLoad) when handling `dbt seed` command. The option will likely speed up loading when your seed files have hundreds of thousands of rows. You can set this seed configuration option in your `project.yml` file, e.g.:
```

Comment:
Do you forsee folks using dbt seed to upload a lot of files? We generally do not recommend using dbt seed for anything larger than say a few thousand rows. This is because every seeded file is version controlled and this is generally not necessary for what they are loading.

Reply:
We have to educate our customers that `dbt seed` is not a general-purpose data load tool and is reserved for data that is required for app logic and must be tracked in git as it changes. Customers still expect us to speed things up in case they want to load large csv files.

If you think I should add verbiage here that enforces the point, please let me know.

Reply:
Yes please! A big call out box warning that this point would be great

Reply:
thank you!